wiki/Beatmaps/zh.md

```diff
@@ -0,0 +1,45 @@
+# 谱面
+
+*请勿与 [难度](/wiki/Beatmaps) 相混淆。*
```

Comment:
the link is different in the original article:
```md
*Not to be confused with [Difficulties](/wiki/Difficulties).*
```

Reply:
sorry,i forgot to correct link when copying